realm/realm-library/src/main/java/io/realm/RealmQuery.java

```diff
@@ -1925,6 +1936,24 @@ public long count() {
         return this;
     }
 
+    /**
+     * This predicate will always match.
+     */
+    public RealmQuery<E> alwaysTrue() {
```

Comment:
is it necessary to expose these as public API? what's the potential use case?

Reply:
I assume this returns all parents when used with `LinkingObjects`?

Reply:
Yes. From what I can tell from the NSPredicate docs, this is equivalent to `1 == 1` or something like that, and is only really useful in a subquery where you need to have _something_ to be semantically correct, i.e.

```
<query> OR _something_
```